### DISCUSSION TITLE
How to ensure consistent display of fonts?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#10503] [General - Components / Directives / etc] [answered]
I built an SPA that displays at some point in a `<div>`

    vider 🍽️

(🍽️ is copied from [Emojipedia](https://emojipedia.org/fork-and-knife-with-plate/) or [Unicode Character Table](https://unicode-table.com/en/1F37D/)).

On my Windows 10 with Chrome, this is displayed as

![image](https://user-images.githubusercontent.com/1345886/131189372-3beef4d4-859b-4fc8-8328-10adf6da27a8.png)

When running the same `dist` content on a Raspberry Pi with [luakit ](https://luakit.github.io/), I see

![image](https://user-images.githubusercontent.com/1345886/131189469-146216e5-1ae2-4df6-bcfb-df6e66291756.png)

The emoji is gone.

Is there a way to ensure it is present in every display - or at least how to **make it display on that other device/browser**? My understanding is that the fonts are provided in `dist/spa/fonts` so they should be available to the browser (?)

**Comment by pikil** [comment, marked as answer]
These are conventional emojis, but not all the browsers on all the operating systems follow them (also, each operating system has got different drawing for these).
It is safer to use web fonts instead if you need the identical look on almost all browsers:

Have a look at #webfont section on this page:
https://quasar.dev/vue-components/icon#webfont-usage

There are plenty of fork-knife looking icons in these webfonts, so you should find something useful which will be looking *exactly* the same :)

**Reply by wsw70** [reply]
Thanks, this is what I ended up doing.

I knew webfonts and `<q-icon>`, the problem is that the text is generated. This means having HTML in the generated text, which means the need to dynamically render it.  This means using `v-html` - but `v-html` will not dynamically render the HTML and therefore icons won't work :-(

I ended up getting rid of the generated text and using webfonts. Thanks again for pushing me back to that direction :)
